src/javascript/app_2/Modules/SmartChart/Components/Markers/marker-line.jsx

```diff
@@ -1,19 +1,38 @@
-import classNames   from 'classnames';
-import { observer } from 'mobx-react';
-import PropTypes    from 'prop-types';
-import React        from 'react';
+import classNames           from 'classnames';
+import { observer }         from 'mobx-react';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import { Icon }             from 'Assets/Common';
+import { IconEndTimeSVG }   from 'Assets/Contract/icon-end-time.jsx';
+import { IconStartTimeSVG } from 'Assets/Contract/icon-start-time.jsx';
 
 const MarkerLine = ({
     label,
     line_style,
+    status,
 }) => (
     <div className={classNames('chart-marker-line__wrapper', `chart-marker-line--${line_style}`)}>
-        <div className='chart-marker-line__label'>{label}</div>
+        { label === 'End Time' &&
```

Comment:
instead of `=== 'End Time' and === 'Start Time'` I'm thinking if we could link these with the labels in `MARKER_TYPES_CONFIG` so if the labels change there -> this won't break

Reply:
will there be a circular import?

Reply:
ill try inject the marker config to props